doc/run_with_mysql.md

```diff
@@ -0,0 +1,43 @@
+# How to connect MySQL with SQLFlow
+
+This tutorial explains how to connect SQLFlow with [MySQL](https://en.wikipedia.org/wiki/MySQL).
+
+## Connect Existing MySQL server
+
+To connect an existed MySQL server instance, we need to configure a `datasource` string in the format of   
+```
+mysql://{username}:{password}@tcp({address})/{dbname}[?param1=value1&...&paramN=valueN]
+```
+
+In the above format,
+- `username` indicates the user name, e.g `root`.
+- `password` indicates the password, e.g. `root`.
+- `address` indicates the ip address and the port number, e.g. `127.0.0.1:3306`.
+- `dbname` indicates the database name, e.g. `iris`.
+- `param1=value1` indicates additional configurations, e.g. `maxAllowedPacket=0`.
+
+Putting these all together, we can construct a data source string like
+```
+mysql://root:root@tcp(127.0.0.1:3306)/iris?maxAllowedPacket=0
+```
+Using the `datasource`, you may launch an all-in-one Docker container by running:  
+```bash
+> docker run --rm -p 8888:8888 sqlflow/sqlflow bash -c \
+"sqlflowserver --datasource='mysql://root:root@tcp(127.0.0.1:3306)/iris?maxAllowedPacket=0' &
+SQLFLOW_SERVER=localhost:50051 jupyter notebook --ip=0.0.0.0 --port=8888 --allow-root --NotebookApp.token=''"
+```
+
+Open [localhost:8888](localhost:8888) through a web browser, and you will find there are many SQLFlow tutorials, e.g. `iris-dnn.ipynb`. Please follow the tutorials and substitute the data for your use.
```

Comment:
It looks \[localhost:8888](localhost:8888) won't be rendered as a link. So, let's just write `localhost:8888` here

Reply:
Sure, thanks.